docs/getting-started-guides/vsphere.md

```diff
@@ -104,10 +104,17 @@ parameters. The guest login for the image that you imported is `kube:kube`.
 Now, let's continue with deploying Kubernetes.
 This process takes about ~20-30 minutes depending on your network.
 
+#### From extracted binary release
 ```sh
-cd kubernetes # Extracted binary release OR repository root
-export KUBERNETES_PROVIDER=vsphere
-cluster/kube-up.sh
+cd kubernetes
+KUBERNETES_PROVIDER=vsphere cluster/kube-up.sh
+```
+
+#### Build from source
+```sh
+cd kubernetes
+make release
```

Comment:
Ok. Its just that `make release` will take quite some time and is not required to be run all the time.